code/modules/power/supermatter/supermatter_process.dm

```diff
@@ -370,6 +370,8 @@
 
 	if(prob(5))
 		supermatter_anomaly_gen(src, FLUX_ANOMALY, rand(5, 10))
+	if(power > POWER_PENALTY_THRESHOLD && prob(5) || prob(1))
+		supermatter_anomaly_gen(src, HALLUCINATION_ANOMALY, rand(5, 10))
```

Comment:
this check doesn't make any sense, this proc will not be allowed to get to this point if power is less then `POWER_PENALTY_THRESHOLD`

Reply:
right, i'll just prob 5 it